core/api/jrpc/jrpc_server_impl.cpp

```diff
@@ -15,11 +15,23 @@ OUTCOME_CPP_DEFINE_CATEGORY(kagome::api, JRpcServerImpl::Error, e) {
   }
   return "Unknown error";
 }
+
+namespace {
+  constexpr const char *kRpcRequestsCount = "kagome_rpc_requests_count";
```

Comment:
It is mildly confusing that a "*Count" variable is not an integer, but a string. Just a side-note.